core/src/main/java/nu/yona/server/goals/entities/Goal.java

```diff
@@ -58,6 +78,23 @@ public ZonedDateTime getCreationTime()
 		return creationTime;
 	}
 
+	public ZonedDateTime getEndTime()
```

Comment:
Please modify the 'add inactivity' algorithm to use the end time, or create a ticket for that.